### DISCUSSION TITLE
Azuracast Workflow: Playlists, Streamers and AutoDJ Settings

### DISCUSSION THREAD

**Discussion by surface2air** [AzuraCast/AzuraCast#4554] [Q&A]
Hi there,

I am new to Azuracast, although I have about a month of using a self-hosted version that was set up by an audio engineer and admin.

My questions are primarily related to work flow regarding Playlists, Streamers and AutoDJ settings:

![2021-09-08-12-34-wsor io](https://user-images.githubusercontent.com/4358732/132574754-9b92aaf3-f68f-452b-826c-73d5957dbcd4.png)

**Select screen capture from PROFILE view.**

**WORKFLOW**
-------------------------------------------------------------------------------------

**PLAYLISTS**

**The self-hosted station using Azuracast is a conduit for a private stream of music audio from a remote server that powers music around the clock.**

I have a 3-hour program I have set-up with a Playlist and schedule for a 3-hour block of time scheduled at the same time each week. I have organized the program's Playlist in a linear format to play the tracks in sequence for the duration.

After scheduling the Playlist for the time slot scheduled, I have been instructed to restart the station on or about the time to initialize the Playlist. When I do so, nothing happens but silence _followed by a recording about Azuracast in a short promo loop._

What is the proper work flow to initialize the Playlist to actually play? 

![2021-09-08-12-40-wsor io](https://user-images.githubusercontent.com/4358732/132575237-a37c72b9-4557-4cf7-bdd2-4ca829495745.png)
**Screen capture from PLAYLISTS view.**

![2021-09-08-12-38-wsor io](https://user-images.githubusercontent.com/4358732/132576796-d0e15054-6eb2-44eb-9edc-f902a3241069.png)
**Screen capture from EDIT PLAYLISTS - BASIC INFO view.**

![2021-09-08-12-39-wsor io](https://user-images.githubusercontent.com/4358732/132575233-a28335cb-1a44-4451-baf1-e2bc317868ff.png)
**Screen capture from EDIT PLAYLISTS - SCHEDULE view.**

----------

**STREAMERS / DJs**
I have set up a Streamer / DJ for myself, having read some documentation that by adding and enabling a Streamer, it will record the Playlist ("Program") during the scheduled timeframe and archive it in the Broadcasts folder.

This does not happen.

How can I record the Playlist I have added to the Schedule?

![2021-09-08-12-36-streamer-dj-account-wsor io](https://user-images.githubusercontent.com/4358732/132574914-1b2c664f-3de6-4b52-9a39-9ee993fb0e37.png)
**Screen capture from STREAMERS / DJ ACCOUNTS LIST view.**

![2021-09-08-12-36-edit-streamer-wsor io](https://user-images.githubusercontent.com/4358732/132575132-0e0eaafe-9e93-4a4d-8ab6-3196094c4b94.png)
**Screen capture from STREAMER / DJ ACCOUNTS > EDIT STREAMER > BASIC INFO view.**

![2021-09-08-12-35-wsor io](https://user-images.githubusercontent.com/4358732/132574926-507e75e6-aa0c-4330-aa77-06cb37818660.png)
**Screen capture from STREAMER / DJ ACCOUNTS > EDIT STREAMER > SCHEDULE view.**

-------

**AUTODJ SERVICE / LIQUIDSOAP**
As part of the workflow instructions I have had, I have been trying to RESTART to enable to grant Liquidsoap to initialize the Playlist I have scheduled. This does not seem to happen.

What are the steps required in the Workflow to initialize a RESTART from the AUTODJ SERVICE?

By station default, the Status currently shows that is: NOT RUNNING.

![2021-09-08-12-41-wsor io](https://user-images.githubusercontent.com/4358732/132574820-621bd170-168e-4c7d-be48-09740384ae89.png)

**Screen capture from AUTODJ SERVICE / LIQUIDSOAP module view.**

**Comment by BusterNeece** [comment]
@surface2air One of the big problems you're likely running into is that you have no "general rotation" playlist, i.e. there is no non-disabled playlist in your library that has no scheduling requirements. You only have one playlist, and it's a scheduled one, which means that our system has nothing to play for the remainder of the time, which causes a fair amount of problems for our AutoDJ as it's basically always looking for something to play, even in non-scheduled blocks; the solution is to create a filler playlist that's general rotation, has no scheduled items, and only has a few tracks (if you prefer) that repeat during non-standard broadcast times instead of the default error message.

Broadcast recordings are also affected by the ability to restart Liquidsoap to apply them and turn them on, so it's possible the fact that Liquidsoap isn't restarting correctly for you is also what caused your broadcast not to be recorded properly.

In order for us to help diagnose the Liquidsoap issue, we would need to see your liquidsoap log, viewable via "Utilities" on the station sidebar menu, then "Log Viewer", then "Liquidsoap log".
